pkg/server/handlers.go

```diff
@@ -196,23 +262,24 @@ func createDeployment(w http.ResponseWriter, r *http.Request) {
 		tempPTS.Annotations = make(map[string]string)
 	}
 
-	if tempJSON.PrivateHosts != nil {
-		tempPTS.Annotations["privateHosts"] = *tempJSON.PrivateHosts
-	}
-
-	if tempJSON.PublicHosts != nil {
-		tempPTS.Annotations["publicHosts"] = *tempJSON.PublicHosts
-	}
+	tempPTS.Annotations["edge/paths"] = composePathsJSON(tempJSON.Paths)
 
 	//If map is empty then we need to make it
 	if len(tempPTS.Labels) == 0 {
 		tempPTS.Labels = make(map[string]string)
 	}
 
-	//Add routable label
-	tempPTS.Labels["routable"] = "true"
+	//Add Labels
+	tempPTS.Labels["edge/app.name"] = tempJSON.DeploymentName
+	tempPTS.Labels["edge/routable"] = "true"
```

Comment:
Same as above.  Also, do you think it makes sense to allow extra annotation/labels to be set by the user?  Being able to set `type: interceptor` or something like that could solve the upcoming interceptor/proxy support.